tt_metal/hw/toolchain/sections.ld

```diff
@@ -26,14 +26,15 @@
 #if defined(TARGET_IERISC)
 #define MEM_PAD MEM_IERISC_KERNEL_PAD
 #else
-#define MEM_PAD MEM_TENSIX_KERNEL_PAD
+#define MEM_PAD 0
 #endif
+/* FW must align to 16 byte boundary so kernel begins aligned to meet noc alignment constraints */
+#define MEM_ALIGN 16
 #else
 #define MEM_PAD 0
+#define MEM_ALIGN 4
```

Comment:
do we need this? from usage below it looks like `MEM_ALIGN` used is guarded under `#ifdef TYPE_FIRMWARE`?

Reply:
one use is, other use is not